openapi.yaml

```diff
@@ -19,14 +19,27 @@ info:
   title: Fulcio
 schemes: [http, https]
 host: fulcio.sigstore.dev
-basePath: /api/v1
+basePath: /
```

Comment:
@bobcallaway push back the basepath because I don't know if we can set a root Path and override the basepath, let me know if this is a good practice or not, thanks in advance